app/models/source.js

```diff
@@ -69,30 +66,27 @@ var methods = Object.assign({
  * @property {string} [totalItemsAvailableFromPagePathTemplate=response.${contentTypePluralCamelName}.count] - String template used to generate object paths to value representing total items available for contentType within pages returned from source
  */
 module.exports = modelFactory.new('Source', {
-    apiVersion: String,
-    authScope: Array,
-    clientId: String,
-    clientSecret: String,
- //   contentTypes: [{ ref: 'ContentType' }],
-    itemDataObjectsFromPagePathTemplate: { type: String, default: 'data' },
-    // this is where you specifiy what you want from source
- //   itemsGetUrlTemplate: { type: String, default: 'https://${host}/${contentTypePluralCamelName}?access_token=${accessToken}&limit=${limit}&offset=${offset}' },
+  apiVersion: String,
```

Comment:
The latest commit looks a lot cleaner than the prior (good work!) but it seems there are still indentation issues, such as here and elsewhere, where I'll note separately. Let's get these resolved too before I review the code for its functional aspects.